rbdmirror: enable rbd rados namespace mirroring (#14701)
rook/rook: Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

See the official rbd mirror documentation on [how to add a bootstrap peer](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#bootstrap-peers).

!!! note

>     Disabling mirroring for the cephblockpool requires disabling mirroring on all the

Reviewer: Is this what you mean?

```suggestion
    Disabling mirroring for the CephBlockPool requires disabling mirroring on all the
```

> !!! note
>     And Disabling the cephblockpool namespace would require to disable mirroring on all the
>     rados namespace present underneath.

Reviewer: I'm having trouble interpreting what this means, and I think users will also be confused. Can we add clarity by specifying the CRD names where appropriate? We could add field names also if needed.

I can make this suggestion, but TBH I am still confused about what the note means in terms of what actions I should take to disable mirroring. 

```suggestion
!!! note
    Disabling the CephBlockPool namespace requires disabling mirroring on all
    CephBlockPoolRadosNamespaces present underneath.
```

What does "Disabling the CephBlockPool namespace" mean? Should this be "Disabling mirroring for the CephBlockPool"? Something else?

I am also wondering whether users should be informed of the inverse. This talks about disabling the feature, but what about enabling the feature? Are there considerations there that need to be noted?

Author: No for enabling we dont need to specify it, that is not restricted or an exception
